storage/bootstrap/bootstrapper/peers/source.go

```diff
@@ -127,14 +137,15 @@ func (s *peersSource) Read(
 	}
 
 	var (
-		lock                sync.Mutex
-		wg                  sync.WaitGroup
-		incrementalWg       sync.WaitGroup
-		incrementalMaxQueue = s.opts.IncrementalPersistMaxQueueSize()
-		incrementalQueue    = make(chan incrementalFlush, incrementalMaxQueue)
-		bopts               = s.opts.ResultOptions()
-		count               = len(shardsTimeRanges)
-		concurrency         = s.opts.DefaultShardConcurrency()
+		lock                    sync.Mutex
```

Comment:
Perhaps we should rename this `resultLock` so it's more clear?

Reply:
done